assets/base.css

```diff
@@ -1191,37 +1191,71 @@ deferred-media {
   position: relative;
 }
 
+.share-button__button {
+  font-size: 1.4rem;
+  display: flex;
+  align-items: baseline;
+  color: rgba(var(--color-link),var(--alpha-link));
+  margin-left: 0;
+  padding-left: 0;
+
+}
+
+.share-button__button, .share-button__fallback button {
+  cursor: pointer;
+  background: var(--color-background);
+  border: none;
+}
+
 .share-button__button .icon-share {
   margin-right: 1rem;
 }
 
 .share-button__fallback {
-  border: 0.1rem solid rgba(var(--color-foreground), 0.1);
   background: rgb(var(--color-background));
-  padding: 2rem;
+  display: flex;
   position: absolute;
-  top: 4rem;
+  top: 3rem;
   left: -0.1rem;
   z-index: 3;
   width: 100%;
-  display: flex;
-  flex-direction: column;
+  box-shadow: 0 0 0 .1rem rgba(var(--color-foreground),.55);
 }
 
```

Comment:
If I am tabbing it closes the sharing pop up when I go to the next item. Is that expected?

https://screenshot.click/09-52-w9hd5-nw0ql.mp4

Reply:
I believe so. It behaves the same as the header. Seems like the `details-disclosure.js` is setup to work that way.